test/puppetlabs/pcp/messaging_test.clj

```diff
@@ -57,8 +57,7 @@
   {:server      "wss://localhost:8143/pcp/"
    :cert        (format "test-resources/ssl/certs/%s.example.com.pem" cn)
    :private-key (format "test-resources/ssl/private_keys/%s.example.com.pem" cn)
-   :cacert      "test-resources/ssl/certs/ca.pem"
-   :type        "demo-client"})
```

Comment:
I guess that verifies it's optional. The type `demo-client`'s not actually asserted anywhere in testing?